lib/ui/src/components/sidebar/SidebarList.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { styled } from '@storybook/theming';
+
+const Heading = styled.span``;
+
+const SidebarList = props => <Heading {...props} />;
+
+export default SidebarList;
```

Comment:
What is this component? I think what you call the `SidebarList` is the `StoriesExplorer` which already is built and has these stories